src/BuildScriptGenerator/DotNetCore/DotnetCorePlatform.cs

```diff
@@ -139,7 +139,8 @@ public PlatformDetectorResult Detect(RepositoryContext context)
             {
                 installBlazorWebAssemblyAOTWorkloadCommand = DotNetCoreConstants.InstallBlazorWebAssemblyAOTWorkloadCommand;
                 manifestFileProperties[ManifestFilePropertyKeys.Frameworks] = "blazor";
-                this.logger.LogInformation("Detected the the following framework(s): blazor");
+                this.logger.LogInformation("Detected the following framework(s): blazor");
```

Comment:
nit: it may be worth creating a `GlobalLogger` class in the future that can log a single message to both Application Insights and STDOUT (although I'm not sure how often it would be used outside of the messages in this PR).

Reply:
Currently these are the only places that are expected to have framework logging. But if we get them for all platforms then I agree we should have a `GlobalLogger`